ahmedheakl/drone-vis
# Feat - YOLOv8 Tracking

Feature - YOLOv8 Tracking
## 📜 Summary
Utilizing the tracking capability of YOLOv8. 

###### Ticket(s): [#yolo-tracking](https://app.clickup.com/yolo-tracking)





- 🔄 **Main changes in this PR**
    - Added YOLOv8 tracking in examples `examples/track_with_yolov8.py`
    - Added GPU support for YOLOv8.



- ⚠️ **Breaking changes**
  - None

## 🚀 How to test this Pull Request

1. Make sure you have your GPU device number registered in Env variable `CUDA_VISIBLE_DEVICES=<GPU-DEVICE_NUMBER>`. If not, please run `export CUDA_VISIBLE_DEVICES=<GPU-DEVICE_NUMBER>` in your terminal
2. Run `py examples/track_with_yolov8.py`

## 📝 Notes for future improvements


- ~
